perceval/backends/core/askbot.py

```diff
@@ -526,12 +526,12 @@ class AskbotCommand(BackendCommand):
 
     BACKEND = Askbot
 
-    @staticmethod
-    def setup_cmd_parser():
+    @classmethod
+    def setup_cmd_parser(cls):
         """Returns the Askbot argument parser."""
 
         parser = BackendCommandArgumentParser(from_date=True,
-                                              archive=True)
+                                              archive=True, categories=cls.BACKEND.CATEGORIES)
```

Comment:
I would put `categories` in a new line to align it with the style followed for the rest of the parameters. Moreover, if we change the definition of `setup_cmd_parser`, it should be also modified in the parent class.

Reply:
Done @sduenas!